### ISSUE TITLE
Different Val Accuracy

### ISSUE THREAD

**Issue by Mechanix12** [catboost/catboost#310]
Hi!
When i train model, it shows me val accuracy near 0.99 (also when use catboost cv). But when use accuracy_score from sklearn.metrics, it shows near 0.7 on the same val set! Where is the truth? :)

**Comment by nikitxskv** [comment]
@Mechanix12 Hi!
Could you provide the sample of code?

**Comment by Mechanix12** [comment]
@nikitxskv 
`X_train, X_validation, y_train, y_validation = train_test_split(X, Y, test_size=0.3, random_state=40, shuffle = True)`
`params = {
    'iterations': 1000,
    'eval_metric': 'Accuracy',
    'logging_level': 'Silent',
    'use_best_model': True
}`
`train_pool = Pool(X_train, y_train, cat_features=cat_features_index)`
`validate_pool = Pool(X_validation, y_validation, cat_features=cat_features_index)`
`model = CatBoostClassifier(**params)`
`model.fit(train_pool, eval_set=validate_pool, plot = True)`
`print(accuracy_score(y_validation, model.predict(X_validation)))`
0.7048947493008128
`ev = pd.DataFrame(model.eval_metrics(validate_pool, metrics=['Accuracy'])['Accuracy'])`
`ev.mean()`
0    0.994218
dtype: float64

**Comment by nikitxskv** [comment]
@Mechanix12 
It is incorrect to get mean from `model.eval_metrics(validate_pool, metrics=['Accuracy'])['Accuracy']`. `eval_metric` method returns metrics values per each iteration. So you should take just last element:
`print(model.eval_metrics(validate_pool, metrics=['Accuracy'])['Accuracy'][-1])`

**Comment by Mechanix12** [comment]
@nikitxskv 
Thanks, but value is near the same:
`print(model.eval_metrics(validate_pool, metrics=['Accuracy'])['Accuracy'][-1])`
0.9945643120786246

**Comment by nikitxskv** [comment]
@Mechanix12 
Do you have the latest CatBoost version?
I couldn't reproduce your error:

```
import catboost
from catboost.datasets import titanic
import numpy as np
from sklearn.model_selection import train_test_split
from sklearn.metrics import accuracy_score

# data preparation
train_df, test_df = titanic()
train_df.fillna(-999, inplace=True)
test_df.fillna(-999, inplace=True)
X, y = train_df.drop('Survived', axis=1), train_df.Survived
categorical_features_indices = np.where(X.dtypes != np.float)[0]
X_train, X_validation, y_train, y_validation = train_test_split(X, y, test_size=0.3, random_state=40, shuffle = True)

# training
params = { 'iterations': 1000, 'eval_metric': 'Accuracy', 'logging_level': 'Silent', 'use_best_model': True, 'random_seed': 0}
train_pool = catboost.Pool(X_train, y_train, cat_features=categorical_features_indices)
validate_pool = catboost.Pool(X_validation, y_validation, cat_features=categorical_features_indices)
model = catboost.CatBoostClassifier(**params)
model.fit(train_pool, eval_set=validate_pool,)
print(accuracy_score(y_validation, model.predict(X_validation))) # 0.8395522388059702
print(model.eval_metrics(validate_pool, metrics=['Accuracy'])['Accuracy'][-1]) # 0.839552238806
```

Could you share your dataset?

**Comment by Mechanix12** [comment]
I found error! I trained MultiClass CatBoostClassifier with labels [0,1,2], but without param `'loss_function':'MultiClass'`.
